game/views/base_view.py

```diff
@@ -10,6 +10,9 @@ def __init__(self, views):
     def switch_to(self, viewname):
         new_view = self.views[viewname]
         new_view.previous_view = self
+
+        if viewname != "game":
```

Comment:
Maybe use an enum object here instead of a string? Or is this relate to some arcade internals?

Reply:
it is the way we have done views. I agree it would be best with an enum. but it's not worth making the alterations now.